### DISCUSSION TITLE
Long-Term-Support end date for 2.x

### DISCUSSION THREAD

**Discussion by mtrezza** [node-fetch/node-fetch#1553] [Q&A] [answered]
The release notes mention Long-Term-Support for 2.x:

> node-fetch from v3 is an ESM-only module - you are not able to import it with require().
>
> If you cannot switch to ESM, please use v2 which remains compatible with CommonJS. Critical bug fixes will continue to be published for v2.

Is there an estimated end date for 2.x LTS?

**Comment by jimmywarting** [comment, marked as answer]
We don't have any estimate, but as long as ppl are willing to report bugs and help fix them by sending in PR, then we will keep accepting PR for bugs and security issues and make new v2.x releases, we understand that not everyone is ready to switch to ESM just yet so we will have a very long date, much longer than NodeJS own release cycle, but don't expect much new features to be added to it. We are now also expecting more and more ppl to start using the new built in fetch provided by NodeJS itself...

**Reply by mtrezza** [reply]
Do you usually also back-port security issues reported for 3.x / 4.x to 2.x? Or does this solely depend on whether someone from the community opens a PR for 2.x?

**Reply by jimmywarting** [reply]
It have happen that we have back-port security issues from v3 to v2 before already, and we might do it again depending on the severity. v2 have been out for a very long time and it's very stable and haven't been touched much for a long while. 

seeing as 14M user use v2 still then we will most definitely backport things
